yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm

```diff
@@ -0,0 +1,260 @@
+	if(H.nutrition < NUTRITION_LEVEL_STARVING + 50)
+		if (H.stat != UNCONSCIOUS && H.stat != DEAD)
+			if(light_level != last_light_level)
+				last_light_level = light_level
+				last_light_message = -STATUS_MESSAGE_COOLDOWN
+				to_chat(H, "<span class='userdanger'>Your internal stores of light are depleted. Find a source to replenish your nourishment at once!</span>")
+			H.take_overall_damage(2, 0)
+	
+
+/datum/species/pod/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(chem.id == "plantbgone")
+		H.adjustToxLoss(3)
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "saltpetre")
+		H.adjustFireLoss(-2.5*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustToxLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "ammonia")
+		H.adjustBruteLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustFireLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "robustharvestnutriment")
+		H.adjustToxLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		for(var/V in H.reagents.reagent_list)//slow down the processing of harmful reagents.
+			var/datum/reagent/R = V
+			if(istype(R, /datum/reagent/toxin) || istype(R, /datum/reagent/drug))
+				R.metabolization_rate = initial(R.metabolization_rate) * 0.5
+
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "left4zednutriment")
+		H.adjustFireLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustToxLoss(1*REAGENTS_EFFECT_MULTIPLIER)
+		if(prob(10))
+			if(prob(95))
+				H.randmutb()
+			else
+				H.randmutg()
+
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "eznutriment")
+		H.adjustToxLoss(-1*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustOxyLoss(-4*REAGENTS_EFFECT_MULTIPLIER)
+		if(H.health < -50)
+			H.adjustOxyLoss(-HUMAN_CRIT_MAX_OXYLOSS)
+
+		if(chem.volume >= 15 && !is_type_in_list(chem, H.reagents.addiction_list))
+			var/datum/reagent/new_reagent = new chem.type()
+			H.reagents.addiction_list.Add(new_reagent)
+
+		for(var/datum/reagent/addicted_reagent in H.reagents.addiction_list)
+			if(istype(chem, addicted_reagent))
+				addicted_reagent.addiction_stage = -15
+
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "diethylamine")
+		if(chem.overdosed)
+			return 0
+
+		if(chem.volume > 20)
+			chem.overdosed = 1
+			chem.overdose_start(H)
+			return 0
+
+		H.adjustBruteLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustFireLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustToxLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustOxyLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "sugar")
```

Comment:
omg this is shitcode, why have a set way for the race to react to sugar, when you could just add diabetes as a disease and give all plant people that. Fucking idiots learn to simplify your code